bin/do_xcf.py

```diff
@@ -186,17 +200,22 @@ def corr_func(p):
     nb = nbs.sum(axis=0)
 
     out = fitsio.FITS(args.out,'rw',clobber=True)
-    head = {}
-    head['RPMIN']=xcf.rp_min
-    head['RPMAX']=xcf.rp_max
-    head['RTMAX']=xcf.rt_max
-    head['Z_CUT_MIN']=xcf.z_cut_min
-    head['Z_CUT_MAX']=xcf.z_cut_max
-    head['NT']=xcf.nt
-    head['NP']=xcf.np
-    head['NSIDE']=xcf.nside
-
-    out.write([rp,rt,z,nb],names=['RP','RT','Z','NB'],header=head)
-    head2 = [{'name':'HLPXSCHM','value':'RING','comment':'healpix scheme'}]
-    out.write([hep,wes,cfs],names=['HEALPID','WE','DA'],header=head2)
+    head = [ {'name':'RPMIN','value':xcf.rp_min,'comment':'Minimum r-parallel'},
+        {'name':'RPMAX','value':xcf.rp_max,'comment':'Maximum r-parallel'},
+        {'name':'RTMAX','value':xcf.rt_max,'comment':'Maximum r-transverse'},
+        {'name':'NP','value':xcf.np,'comment':'Number of bins in r-parallel'},
+        {'name':'NT','value':xcf.nt,'comment':'Number of bins in r-transverse'},
+        {'name':'Z_CUT_MIN','value':xcf.z_cut_min,'comment':'Minimum redshift of pairs'},
+        {'name':'Z_CUT_MAX','value':xcf.z_cut_max,'comment':'Maximum redshift of pairs'},
```

Comment:
@londumas could you change `Z_CUT_MIN` and `Z_CUT_MAX` to `ZCUTMIN` and `ZCUTMAX` ? keyword names in .fits must be 8 characters or less...